exchanges/gateio/gateio_ws_option.go

```diff
@@ -432,17 +429,17 @@ func (g *Gateio) processOptionsOrderPushData(data []byte) error {
 			AccountID:      resp.Result[x].User,
 		}
 	}
+	g.Websocket.DataHandler <- orders
```

Comment:
This is not being handled in `websocketroutine_manager.go`

Reply:
Please revert and send a slice **then** add the slice handling to `websocketroutine_manager.go` for future implementations as this is much more efficient. Please revert all these types of changes and implement.